.vscodeignore

```diff
@@ -1,17 +1,44 @@
-.vscode/**
-.vscode-test/**
-out/test/**
-out/**/*.map
-out/**/*.ts
-src/**
-gif/**
-hooks/**
-test/**
-docs/**
+.vscode
+.vscode-test
+!node_modules/keytar
```

Comment:
Can you add a comment for these vscode ignore files/folders? For example, some are for customized webpack, some are just for CI/CD. This can help developers understand when they should add files in this .vscodeignore file.

Reply:
OK.